src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Windows.targets

```
@@ -33,6 +35,7 @@ The .NET Foundation licenses this file to you under the MIT license.
     <ItemGroup>
       <NativeLibrary Include="$(IlcSdkPath)$(BootstrapperName)$(LibrarySuffix)" />
       <NativeLibrary Include="$(IlcSdkPath)$(FullRuntimeName)$(LibrarySuffix)" />
+      <NativeLibrary Include="$(IlcSdkPath)$(EventPipeName)$(LibFileExt)" />
```

This needs to be:

```suggestion
      <NativeLibrary Include="$(IlcSdkPath)$(EventPipeName)$(LibrarySuffix)" />
```

Once you do that, it will probably start failing some legs because we need a `.GuardCF` version of the static libraries to support control flow guard.

A GuardCF flavor can be defined like this:

https://github.com/dotnet/runtime/blob/a34291586e5f82a98f6b5a2e9134cee96f7fc306/src/coreclr/nativeaot/Bootstrap/base/CMakeLists.txt#L11-L15